addon/services/cookies.js

```diff
@@ -173,9 +173,10 @@ export default Service.extend({
   },
 
   _filterCachedFastBootCookies(fastBootCookies) {
-    let { path: requestPath } = get(this._fastBoot, 'request');
+    let { path: requestPath } = this._fastBoot.request;
 
     // cannot use deconstruct here
+    // eslint-disable-next-line ember/no-get
     let host = get(this._fastBoot, 'request.host');
```

Comment:
this one can't be converted yet, unless we drop support for Ember 3.0 first

Reply:
Why not? Could we not just
```suggestion
    let request = this._fastBoot ? this._fastBoot.request : null;
    let host = request ? request.host : null;
```

Reply:
because the Ember 3.0 test case fails, complaining about host being a computed property and not being able to read it like that. haven't dug into the details that much, but it didn't seem like there would be an easy way around it.

Reply:
yeah, that makes sense